test/fail_compilation/retscope2.d

```diff
@@ -130,7 +130,7 @@ fail_compilation/retscope2.d(721): Error: returning `s.get1()` escapes a referen
 
 struct S700
 {
-    @safe S700* get1() return scope
+    @safe S700* get1() return
```

Comment:
Instead of `return scope`, rewrite as `scope return`.

Reply:
Okay. I didn't add scope back because it gets stripped anyway since S700 has no (pointer) members.

Reply:
Done